fluidBuild.config.cjs

```diff
@@ -149,6 +149,7 @@ module.exports = {
 				"package.json",
 				"packages/test/test-service-load/package.json",
 			],
+			"npm-package-json-clean-script": ["tools/markdown-magic"],
```

Comment:
Nit: Can we add a comment explaining why this is being excluded? Will make it easier to evaluate whether or not this is still necessary in the future.

Reply:
done.